packages/core/src/browser/style/tabs.css

```diff
@@ -47,6 +46,7 @@
   flex: none;
   align-items: flex-end;
   z-index: 1;
+  cursor: pointer;
```

Comment:
How would that behave if the `p-TabBar` is in another `data-orientation`? (Does that ever happen?)

Reply:
Thank you, I think you are right, it should be `pointer` no matter what `data-orientation` is.
I will modify it soon.

Reply:
For reference, that's how I did it: https://github.com/theia-ide/theia/pull/5541/commits/f11ebc248afc4fd6de7cf0b8e5232d6f2ee2872b

Reply:
Thank you a lot.

Reply:
I'll re-review :)